packages/launch/src/launch/styles.scss

```diff
@@ -6,6 +6,16 @@
 @import '~@automattic/onboarding/styles/variables';
 @import '~@automattic/onboarding/styles/mixins';
 
+$padding-h-mobile: 20px;
+$padding-h-small: 32px;
+$padding-h-medium: 48px;
+$padding-h-large: 60px;
+$padding-h-wide: 80px;
```

Comment:
Shouldn't these go in the variables file you've created?

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/47574/commits/1b6f9c4f6b83077da6fa1742830f800daf3addc5